.prettierrc.json

```diff
@@ -0,0 +1,10 @@
+{
+    "arrowParens": "always",
+    "bracketSpacing": true,
+    "jsxBracketSameLine": true,
+    "plugins": ["prettier-plugin-tailwindcss"],
+    "semi": true,
+    "singleQuote": true,
```

Comment:
I see prettier is configured for single quotes but the entire project is actually using double quotes.
This might be caused by a different config in eslint and might lead to some developers not being able to commit because VS Code formats in one way, but commit time linting expects a different way.

If I remember correctly there's some [ready made configs](https://prettier.io/docs/en/integrating-with-linters.html) that are meant to solve such configuration collisions.

Reply:
Added `quote_type=single` to `editorconfig` and also added `[eslint-config-prettier](https://github.com/prettier/eslint-config-prettier)`. Thanks for the tip!

Reply:
Markdown protip: if you want to have a link formatted as code, you have to wrap the backticks in brackets, not the other way around.
i.e. ``[`eslint-config-prettier`](https://github.com/prettier/eslint-config-prettier)`` => [`eslint-config-prettier`](https://github.com/prettier/eslint-config-prettier)